Cauldron/Assets/Scripts/UI/Menu/MainMenu.cs

```diff
@@ -50,11 +66,12 @@ private void HideContinueButton()
             continueGame.gameObject.SetActive(false);
         }
 
-        private void NewGameClick()
+        private async void NewGameClick()
         {
+            await fade.FadeIn(fadeNewGameDuration, 1);
```

Comment:
StartNewGame() is a more suitable place for this fade, because here we can do some logic to abort the operation.